### ISSUE TITLE
React example

### ISSUE THREAD

**Issue by barbier** [wejs/we#166]
Which type of example should be created?

**Comment by albertosouza** [comment]
Hi @barbier ,
You want make a React + we.js example?

**Comment by barbier** [comment]
Yes! It's on the roadmap, so I think it would be nice to contribute with the project :)

**Comment by albertosouza** [comment]
@barbier Great!
You can test the Vue.js and We.js example: https://github.com/albertosouza/twitter-prizewinner
Where: 
1- All Vue.js code generated from Vue.js generator are at client folde and its build are in client/dist folder.
2- The we.js have all api and integration with external services (twitter) in we.js project
3- Vue.js production code are served with express static midleware with code: https://github.com/albertosouza/twitter-prizewinner/blob/master/plugin.js#L67

Bônus:
1- Use the we.js heroku generator to generate code for deploy on heroku
2- Add the heroku deploy button in README.md

...
Callme on gmail at alberto.souza.99@gmail.com ... i want help with this example.
